concourse/scripts/pipecleaner.py

```diff
@@ -171,7 +177,8 @@ def check_pipeline(self, data):
                         shellcheck_dialects =  ['sh', 'bash', 'dash', 'ksh']
                         if item['config']['run']['path'] in shellcheck_dialects:
                             exitcode, output = self.call_shellcheck(item['config']['run']['path'],
-                                                                    item['config']['run']['args'])
+                                                                    item['config']['run']['args'],
+                                                                    item['config'].get('params', {}))
```

Comment:
The logic here isn't quite right.  You can also specify params at the task level. https://concourse.ci/task-step.html#task-params